xml/System.Diagnostics/StackFrameExtensions.xml

```diff
@@ -46,10 +46,11 @@
         <Parameter Name="stackFrame" Type="System.Diagnostics.StackFrame" RefType="this" />
       </Parameters>
       <Docs>
-        <param name="stackFrame">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="stackFrame">A stack frame.</param>
+        <summary>Returns a pointer to the base address of the native just-in-time (JIT)-compiled image that this stack frame is executing.</summary>
+        <returns>This method always returns <see cref="F:System.IntPtr.Zero" />.</returns>
```

Comment:
I believe this is not entirely true for .NET Native case. @trylek is that right?